forge/forge/python_codegen.py

```diff
@@ -76,6 +76,8 @@ def pytorch_df_str_from_str(df: str, name):
         return "torch.int16"
     elif df == "int64":
         return "torch.int64"
+    elif df == "uint1":
+        return "torch.bool"
     else:
         logger.warning(f"Invalid data format: {df} for constant/parameter '{name}', defaulting to float32")
```

Comment:
Noted, but can we expand on idea offline during planning of dataformats effort

Reply:
Ofc, we should create a brief propose doc that will capture what we're going to do, and what it's going to impact potentially. This will be a great mediator in our further planning of for data formats effort